.packit.yaml

```diff
@@ -47,3 +47,49 @@ jobs:
       - fedora-all
       - rhel-8
       - rhel-9
+
+  - job: tests
+    trigger: pull_request
+    identifier: "unit/centos-stream"
+    targets:
+      - centos-stream-8
+      - centos-stream-9
+    labels:
+      - unit
+    tf_extra_params:
+      environments:
+        - artifacts:
+            - type: repository-file
+              id: https://copr.fedorainfracloud.org/coprs/g/yggdrasil/latest/repo/centos-stream-$releasever/group_yggdrasil-latest-centos-stream-$releasever.repo
```

Comment:
It seems that we have same problem with rhel 8/9 tests as we had with rch in this PR: https://github.com/RedHatInsights/rhc/pull/107 There is workaround for this issue: https://github.com/RedHatInsights/rhc/pull/107/files#diff-0673ea92d72beb0d23ba537e1e83854385c56c702ef9abbed890b4a31ed4ece0R50

Reply:
I'm not clear what the workaround you're proposing is. Is it to carry one target per job, instead of having both `centos-stream-8` and `centos-stream-9` in the same job?